model/activist.go

```diff
@@ -1131,12 +1146,14 @@ func UpdateActivistData(db *sqlx.DB, activist ActivistExtra, userEmail string) (
 	// TODO: make this work for other chapters too, maybe based on Chapters table mailing list info?
 	// or maybe just passing chapter Name or ID to the signup service?
 	if origActivist.ChapterID == 47 {
-		activistInfoChanged := activist.Name != origActivist.Name ||
+		activistAddressChanged := activist.StreetAddress != origActivist.StreetAddress ||
```

Comment:
looks like StreetAddress is not used by the mailing list, so this should not trigger a call to update the mailing list.